### PR TITLE
HotFix - attempt to fix bulkrax importer issue

### PR DESCRIPTION
**Pull request by ShanaLMoore**
Manually, uploading a work seems to be OK but there are issues getting all the jobs to run with Bulkrax.
This PR removed the sidekiq.yml file, which I think may have been the cause of some of the confusion. 

Prior to this PR, we couldn't get certain jobs to run at all. 

# Story

Approve per testing results by @laritakr

### CODE REVIEW

**Review by laritakr — APPROVED** [review]
I'd like to understand when jobs need to be listed and prioritized to make sure we have the right ones. But this change made significant progress in eliminating difficulties with the jobs.

File: config/initializers/good_job.rb [hunk 1]
```diff
@@ -10,4 +10,18 @@
...
+Bulkrax::ImportWorkJob.priority = -5
+Bulkrax::ImporterJob.priority = -20
+Bulkrax::CreateRelationshipsJob.priority = -10
+CharacterizeJob.priority = 30
+ContentDepositEventJob.priority = -50
+ContentUpdateEventJob.priority = -50
+CreateDerivativesJob.priority = 40
+Bulkrax::ScheduleRelationshipsJob.priority = 50
+Hyrax::GrantEditToMembersJob.priority = 10
+IngestJob.priority = 10
```
Comment (alishaevn) [inline config/initializers/good_job.rb:27]:
```suggestion
ApplicationJob.priority = 0
AttachFilesToWorkJob.priority = -1
Bulkrax::CreateRelationshipsJob.priority = -10
Bulkrax::ImporterJob.priority = -20
Bulkrax::ImportWorkJob.priority = -5
Bulkrax::ScheduleRelationshipsJob.priority = 50
CharacterizeJob.priority = 30
ContentDepositEventJob.priority = -50
ContentUpdateEventJob.priority = -50
CreateDerivativesJob.priority = 40
Hyrax::GrantEditToMembersJob.priority = 10
IngestJob.priority = 10
```

Comment (alishaevn) [reply config/initializers/good_job.rb:27]:
```suggestion
Bulkrax::ScheduleRelationshipsJob.priority = 50
CreateDerivativesJob.priority = 40
CharacterizeJob.priority = 30
Hyrax::GrantEditToMembersJob.priority = 10
IngestJob.priority = 10
ApplicationJob.priority = 0
AttachFilesToWorkJob.priority = -1
Bulkrax::ImportWorkJob.priority = -5
Bulkrax::CreateRelationshipsJob.priority = -10
Bulkrax::ImporterJob.priority = -20
ContentDepositEventJob.priority = -50
ContentUpdateEventJob.priority = -50
```

Comment (alishaevn) [reply config/initializers/good_job.rb:27]: the first suggestion is to list these alphabetically. the second, is listing in order of priority. I'm fine with either.

**Review by alishaevn — CHANGES_REQUESTED** [review]

**Review by alishaevn — APPROVED** [review]

File: config/initializers/good_job.rb (hunk 1 above)
Comment (ShanaLMoore) [reply config/initializers/good_job.rb]: great idea! thanks :)

**Pull request merged**